TIL/daily/250317.md

```
@@ -0,0 +1,244 @@
+  
+  // Map의 entries 출력 (Python의 items()와 유사)
+  for (var entry in scores.entries) {
+    print('${entry.key}: ${entry.value}');
+  }
+  
+}
+```
+
+```dart
+  void main() {
+	  // Map value에만 dynamic 사용 가능
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)